kogito-kafka-quickstart-springboot/README.md

```diff
@@ -59,28 +118,66 @@ java -jar target/kogito-kafka-quickstart-sprintboot-{version}.jar
 
 To make use of this application it is as simple as putting a message on `travellers` topic with following content 
 
+* To examine ProcessedTravellers topic and verify upcoming messages will be processed
+
+```
+kafka_2.11-2.4.0/bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic processedtravellers 
+```
+
+* Send Message to Topic
+```
+kafka_2.11-2.4.0/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
 ```
-{ 
-"firstName" : "Jan", 
-"lastName" : "Kowalski", 
-"email" : "jan.kowalski@example.com", 
-"nationality" : "Polish"
-}
 
+Content (cloud event format)
+```
+{
+  "specversion": "0.3",
+  "id": "21627e26-31eb-43e7-8343-92a696fd96b1",
+  "source": "",
+  "type": "VisaApplicationsMessageDataEvent_8", 
+  "time": "2019-10-01T12:02:23.812262+02:00[Europe/Warsaw]",
+  "data": { 
+	"firstName" : "Jan", 
+	"lastName" : "Kowalski", 
+	"email" : "jan.kowalski@example.com", 
+	"nationality" : "Polish"
+	}
+}
+```
+One liner
+```
+{"specversion": "0.3","id": "21627e26-31eb-43e7-8343-92a696fd96b1","source": "","type": "VisaApplicationsMessageDataEvent_8", "time": "2019-10-01T12:02:23.812262+02:00[Europe/Warsaw]","data": { "firstName" : "Jan", "lastName" : "Kowalski", "email" : "jan.kowalski@example.com", "nationality" : "Polish"}}
 ```
 
 this will then trigger the successful processing of the traveller and put another message on `processedtravellers` topic.
 
 To take the other path of the process put following message on `travellers` topic
 
+* Send Message to Topic
 ```
-{ 
-"firstName" : "John", 
-"lastName" : "Doe", 
-"email" : "john.doe@example.com", 
-"nationality" : "American"
+kafka_2.11-2.4.0/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
+```
+With the following content (Cloud Event Format)
+```
+{
+  "specversion": "0.3",
+  "id": "31627e26-31eb-43e7-8343-92a696fd96b1",
+  "source": "",
+  "type": "VisaApplicationsMessageDataEvent_8", 
```

Comment:
type should be `TravellersMessageDataEvent_3` (see Quarkus)

Reply:
fixed @radtriste